run_route_scripts/gen_requests.py

```diff
@@ -22,20 +25,53 @@ def get_random_locs(yc, xc, radius, count):
         y = r * math.sin(c)
         x += xc
         y += yc
-        locs.append({'lon':round(x, 6),'lat':round(y, 6)});
+        locs.append({'lon':round(x, 6),'lat':round(y, 6),'type':"break"});
     return locs
-
+    
+def get_random_intermediate_locs(yc, xc, radius, count, intermediate):
+    intermediate_locs = []
+    rads = radius / 110567.0
+    for i in range(0, count):
+        a = float(random.uniform(0.0, 1.0))
+        b = float(random.uniform(0.0, 1.0))
+        r = rads * math.sqrt(a)
+        c = 2 * math.pi * b
+        x = r * math.cos(c)
+        y = r * math.sin(c)
+        x += xc
+        y += yc
+        intermediate_locs.append({'lon':round(x, 6),'lat':round(y, 6),'type':intermediate});
+    return intermediate_locs
```

Comment:
this function is identical to the one above, why duplicate? just pass the location type as a parameter right?